hash_tree_root.go

```diff
@@ -32,7 +32,12 @@ func HashTreeRoot(val interface{}) ([32]byte, error) {
 	if err != nil {
 		return [32]byte{}, newHashError(fmt.Sprint(err), rval.Type())
 	}
-	output, err := sszUtils.hasher(rval)
+	var output [32]byte
+	if useCache {
```

Comment:
is useCache a flag?

How would anyone change that?

Reply:
i was a flag in the old version. anyway i need it as a testing param. i think it can be on by default

Reply:
Should we remove it and make it always enabled?

Reply:
how will we be able to test it?